services/user-management-services.yaml

```
@@ -384,36 +426,42 @@ paths:
           schema:
             type: string
             format: uuid
+        - name: workShiftId
+          required: true
+          in: path
+          description: work shift's ID
+          schema:
+            type: string
+            format: uuid
       requestBody:
         required: true
         content:
           application/json:
             schema:
-              $ref: "#/components/schemas/WorkEvent"
+              $ref: "#/components/schemas/WorkShiftTimeEntry"
       responses:
         "201":
-          description: Created work event
+          description: Created employee work shift time entry
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/WorkEvent"
+                $ref: "#/components/schemas/WorkShiftTimeEntry"
         "default":
           description: Invalid request was sent to the server
           content:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
 
-  "/v1/employees/{employeeId}/workEvents/{workEventId}":
+  "/v1/employees/{employeeId}/workShifts/{workShiftId}/timeEntries/{timeEntryId}":
     get:
-      operationId: findEmployeeWorkEvent
-      summary: Find an employee's work event.
-      description: Finds an employee's work event by ID.
+      operationId: getWorkShiftTimeEntry
```

by naming convention operations that return just one item are named findXXX

```suggestion
      operationId: findWorkShiftTimeEntry
```